src/wymeditor/core.js

```diff
@@ -775,17 +775,30 @@ jQuery.fn.wymeditor = function (options) {
         preInit: null,
         preBind: null,
         postInit: null
+    }
+
+    mergedOptions = jQuery.extend(defaultOptions, options);
 
-    }, options);
+    // If an iframeHtml option wasn't included, default to using the iframeHtml
+    // defined by the skin.
+    if (typeof options.iframeHtml !== "undefined") {
```

Comment:
Oh, oops. That is the reverse logic:
![screenshot_2015-04-24_13-04-46](https://cloud.githubusercontent.com/assets/635591/7316769/8056fe4a-ea82-11e4-842a-c17126bfa2f0.png)

So I would write
```js
if (options.iframeHtml) {
// ...
}
```